package/Dockerfile

```diff
@@ -1,26 +1,26 @@
-FROM ubuntu:18.04 as intermediate
+FROM registry.suse.com/suse/sle15:15.3 as intermediate
 
 #Label the image for cleaning after build process
 LABEL stage=intermediate
 
-RUN apt-get update \
-    && apt-get install -y \
+RUN zypper --non-interactive update \
+    && zypper --non-interactive install \    
```

Comment:
Need to remove extra whitespace at end of line 7

Reply:
Removed the extra whitespace